main.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	partner "github.com/a-bigelow/cdk-eventbridge-partner-processors-go"
+	"github.com/aws/aws-cdk-go/awscdk/v2"
+	"github.com/aws/aws-cdk-go/awscdk/v2/awsevents"
+	"github.com/aws/constructs-go/constructs/v10"
+	"github.com/aws/jsii-runtime-go"
+)
+
+type ClusterStackProps struct {
+	awscdk.StackProps
+}
+
+func NewClusterStack(scope constructs.Construct, id string, props *ClusterStackProps) awscdk.Stack {
+	var sprops awscdk.StackProps
+	if props != nil {
+		sprops = props.StackProps
+	}
+	stack := awscdk.NewStack(scope, &id, &sprops)
+
+	// The code that defines your stack goes here
+	eventProps := awsevents.EventBusProps{EventBusName: jsii.String("name")}
+
+	eventBus := awsevents.NewEventBus(stack, jsii.String("id"), &eventProps)
+
+	secret := "VerySecretString"
+	partnerProcessor := partner.GithubEventProcessor{
+		EventBus:                       eventBus,
+		WebhookSecret:                  secret,
+		LambdaInvocationAlarmThreshold: 2000,
+	}
+
+	_ = partner.NewGitHubEventProcessor(stack, jsii.String("id"), partnerProcessor)
+
+	return stack
+}
+
+func main() {
+	app := awscdk.NewApp(nil)
+
+	NewClusterStack(app, "ClusterStack", &ClusterStackProps{
+		awscdk.StackProps{
+			Env: env(),
+		},
+	})
+
+	app.Synth(nil)
+}
+
+// env determines the AWS environment (account+region) in which our stack is to
+// be deployed. For more information see: https://docs.aws.amazon.com/cdk/latest/guide/environments.html
+func env() *awscdk.Environment {
+	// If unspecified, this stack will be "environment-agnostic".
+	// Account/Region-dependent features and context lookups will not work, but a
+	// single synthesized template can be deployed anywhere.
+	//---------------------------------------------------------------------------
+	return nil
+
+	// Uncomment if you know exactly what account and region you want to deploy
+	// the stack to. This is the recommendation for production stacks.
+	//---------------------------------------------------------------------------
+	// return &awscdk.Environment{
+	//  Account: jsii.String("123456789012"),
+	//  Region:  jsii.String("us-east-1"),
+	// }
+
+	// Uncomment to specialize this stack for the AWS Account and Region that are
+	// implied by the current CLI configuration. This is recommended for dev
+	// stacks.
+	//---------------------------------------------------------------------------
+	// return &awscdk.Environment{
+	//  Account: jsii.String(os.Getenv("CDK_DEFAULT_ACCOUNT")),
+	//  Region:  jsii.String(os.Getenv("CDK_DEFAULT_REGION")),
+	// }
+}
```

Comment:
We can axe this boilerplate -- peeps will handle their env however they want, it's not really in scope for this repo.